ground/uavobjgenerator/uavobjgenerator.pro

```diff
@@ -7,10 +7,12 @@ macx {
 
 cache()
 
-# Unfortunately this is ineffective on OSX, due to
-# https://bugreports.qt.io/browse/QTBUG-39417
-# Should use it once upstream defect resolved
-QMAKE_CXX=$$(CCACHE_BIN) g++
+*-g++* {
+    # Unfortunately this is ineffective on OSX, due to
```

Comment:
Can remove this comment too if you want (since ccache is now used on OSX post-qt upgrades and other fixes)